src/app/core/component.ts

```diff
@@ -451,13 +456,12 @@ export default class Component {
         this.xspan = this._xspan;
         this.yspan = this._yspan;
 
-        const params = {};
-        if (entity === "TEST MINT") {
-            console.warn("Found legacy invalid entity string", entity);
-            entity = name.split("_")[0]; // '^.*?(?=_)'
+        const params = json.params
+
+        entity = name.split("_")[0]; // '^.*?(?=_)'
```

Comment:
you shouldn't have to do split or anything, entity should just be taken from the entity property of the json